lib/main_containers.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: module loader of container tests
+# Maintainer: qa-c@suse.de
+
+package main_containers;
+use base 'Exporter';
+use Exporter;
+use utils;
+use version_utils;
+use main_common qw(loadtest boot_hdd_image);
+use testapi qw(check_var get_required_var get_var);
+use Utils::Architectures;
+use strict;
+use warnings;
+
+
+our @EXPORT = qw(
+  is_container_test
+  load_container_tests
+);
+
+sub is_container_test {
+    get_var('CONTAINER_RUNTIME') ? return 1 : return 0;
+}
+
+sub load_tests_podman_image {
+    loadtest 'containers/podman_image';
+}
+
+sub load_tests_docker_image {
+    loadtest 'containers/docker_image';
+    loadtest 'containers/container_diff';
+}
+
+sub load_tests_podman_common {
+    unless (is_sle('<=15')) {
+        loadtest 'containers/podman';
+        loadtest 'containers/podman_image';
+        loadtest 'containers/buildah_podman';
+        loadtest 'containers/rootless_podman';
+        loadtest 'containers/podman_3rd_party_images';
+    }
+}
+
+sub load_tests_docker_common {
+    loadtest 'containers/docker';
+    unless (is_sle("<=15") and is_aarch64) {
```

Comment:
```suggestion
    if (is_sle(">15")  && !is_aarch64) {
```

Reply:
Your approach doesn't fit with the scenario.
these 2 packages are available for ALL the cases EXCEPT (aarch64, <=15)

https://confluence.suse.com/display/ENGCTNRSTORY/Level+Test+Plan#LevelTestPlan-PackagesfromContainersModule

Reply:
Ups, yeah it should be `or` 
In the end I think your logic looks more sane :) so I revert my advice!